NDB.Covid19/NDB.Covid19.Droid/Views/DailyNumbers/DailyNumbersActivity.cs

```diff
@@ -106,11 +115,12 @@ private void Init()
             _dailyNumbersSubHeaderStatistics.SetAccessibilityDelegate(AccessibilityUtils.GetHeadingAccessibilityDelegate());
             _dailyNumbersSubHeaderVaccinations.SetAccessibilityDelegate(AccessibilityUtils.GetHeadingAccessibilityDelegate());
             _dailyNumbersSubHeaderSmittestopp.SetAccessibilityDelegate(AccessibilityUtils.GetHeadingAccessibilityDelegate());
+        }
 
-            _closeButton = FindViewById<ImageView>(Resource.Id.daily_numbers_back_button);
-            _closeButton.Click += new StressUtils.SingleClick(OnCloseBtnClicked).Run;
-            _closeButton.ContentDescription = BACK_BUTTON_ACCESSIBILITY_TEXT;
-            _closeButton.SetBackgroundResource(LayoutUtils.GetBackArrow());
+        protected override void OnResume()
+        {
+            base.OnResume();
+            RequestFHIDataUpdate(UpdateUI);
```

Comment:
Please ensure that the update of the UI happens on Main thread to prevent crashes.